README.md

```diff
@@ -313,6 +313,80 @@ Some Serilog packages require a reference to a logger configuration object. The
 },
 ```
 
+### Destructuring
+
+Destructuring means extracting pieces of information from an object and create properties with values; Serilog offers the [@ destructuring operator](https://github.com/serilog/serilog/wiki/Structured-Data#preserving-object-structure). In case there is a need to customize the way log events are serialized (e.g., hide property values or replace them with something else), one can define several destructuring policies, like this:
+
+```yaml
+"Destructure": [
+  {
+    "Name": "With",
+    "Args": {
+      "policy": "MyFirstNamespace.FirstDestructuringPolicy, MyFirstAssembly"
+    }
+  },
+  {
+    "Name": "With",
+    "Args": {
+      "policy": "policy": "MySecondNamespace.SecondDestructuringPolicy, MySecondAssembly"
+    }
+  },
+   {
+    "Name": "With",
+    "Args": {
+      "policy": "policy": "MyThirdNamespace.ThirdDestructuringPolicy, MyThirdAssembly"
+    }
+  },
+],
+```
+
+This is how the first destructuring policy would look like:
+
+```csharp
+namespace MyFirstNamespace
+{
+  public class MyDto
+  {
+    public int Id {get; set;}
+    public int Name {get; set;}
+  }
+
+  public class FirstDestructuringPolicy : IDestructuringPolicy
+  {
+    public bool TryDestructure(object value, ILogEventPropertyValueFactory propertyValueFactory, out LogEventPropertyValue result)
```

Comment:
Some of these lines will cause scrollbars to show up in the README - would it be possible to manually wrap them?